content/docs/command-reference/exp/init.md

```diff
@@ -0,0 +1,137 @@
+# exp init
+
+Initializes experiments.
```

Comment:
p.s applies here too.

Reply:
OK so for the short description, how about something like

> Helper command to setup and run experiments from scratch.

Or something more specific like

> Codify a project variation using [DVC metafiles] and run it as a controlled [experiment]